instances/widgets.treasury-factory.near/widget/pages/settings/MembersEditor.jsx

```diff
@@ -194,6 +204,45 @@ function cleanInputs() {
   setRoles([]);
 }
 
+function detectImpactedPermissions() {
+  const { updatedPolicy } = updateDaoPolicy(
+    new Map(roles.map((role) => [role.title, role.value]))
+  );
+  const permissionsImpactions = updatedPolicy.roles.map((role) => {
+    const votersSize = role.kind.Group.length;
```

Comment:
please add a check, it's possible Group is not a array when the kind is `everyone`, [example](https://github.com/near-daos/sputnik-dao-contract/blob/278adc713e795f95a6da4d3007c7e03e8120f153/README.md?plain=1#L182)